pkg/collector/corechecks/containers/containerd.go

```diff
@@ -296,6 +298,13 @@ func computeHugetlb(sender aggregator.Sender, huge []*cgroups.HugetlbStat, tags
 	}
 }
 
+func computeUptime(sender aggregator.Sender, ctn containers.Container, currentTime time.Time, tags []string) {
+	uptime := currentTime.Sub(ctn.CreatedAt).Seconds()
+	if uptime > 0 {
+		sender.Gauge("containerd.uptime", uptime, "", tags)
```

Comment:
maybe call it containerd.container.uptime? Containerd.uptime makes me think of the uptime of containerd itself. Same for docker and cri

Reply:
I do agree, though it was to be consistent with the metrics we already have containerd.cpu, containerd.mem, etc. which are obviously not for containerd itself and should be called containerd.container.cpu IMHO.